tg_bot/modules/sql/welcome_sql.py

```diff
@@ -22,6 +22,7 @@ class Welcome(BASE):
     leave_type = Column(Integer, default=Types.TEXT.value)
 
     clean_welcome = Column(BigInteger)
+    welcome_mute = Column(Boolean, default=False)
```

Comment:
Same as previous PR. what's the SQL for adding this row.

Reply:
I have a doubt about that. should I create a query that creates the column? can you give me an example? I simply delete the previous DB and do the migration manually. That seems, it's not good.